source/index.txt

```diff
@@ -19,7 +19,7 @@ What is Relational Migrator
    |
 
    .. button:: Download Relational Migrator
-      :uri: http://mongodb.com/try/download/relational-migrator
+      :uri: https://mongodb.com/try/download/relational-migrator
```

Comment:
You can ignore this, hopefully it fixes the external link icon on the download button.